generator/generate-libraries.sh

```diff
@@ -26,41 +26,36 @@ fi
 readonly BAZEL_BIN=${BAZEL_BIN:-/usr/local/bin/bazel}
 readonly BAZEL_OUTPUT_BASE=$("${BAZEL_BIN}" info output_base)
 readonly BAZEL_BIN_DIR=$("${BAZEL_BIN}" info bazel-bin)
-readonly BAZEL_DEPS_GOOGLEAPIS_HASH=$("${BAZEL_BIN}" query 'kind(http_archive, //external:com_google_googleapis)' --output=build | sed -n 's/^.*strip_prefix = "googleapis-\(\S*\)"\,.*$/\1/p')
+readonly BAZEL_DEPS_GOOGLEAPIS_HASH=$("${BAZEL_BIN}" query \
+  'kind(http_archive, //external:com_google_googleapis)' --output=build |
+  sed -n 's/^.*strip_prefix = "googleapis-\(\S*\)"\,.*$/\1/p')
 
 io::log_yellow "Build protoc binary"
 ${BAZEL_BIN} build @com_google_protobuf//:protoc
 
-io::log_yellow "Build microgenerator plugin"
-${BAZEL_BIN} build //generator:protoc-gen-cpp_codegen
-
-# "year product_path proto_path_in_googleapis_repo"
-product_path_proto_path_to_generate=(
-  "2021 google/cloud/bigquery google/cloud/bigquery/storage/v1/storage.proto"
-  "2020 google/cloud/iam google/iam/credentials/v1/iamcredentials.proto"
-  "2021 google/cloud/logging google/logging/v2/logging.proto"
-)
+io::log_yellow "Build microgenerator standalone executable"
+${BAZEL_BIN} build //generator:google-cloud-cpp-codegen
 
 io::log_yellow "Run protoc and format generated .h and .cc files:"
 cd "${PROJECT_ROOT}"
-for proto in "${product_path_proto_path_to_generate[@]}"; do
-  read -r -a tuple <<<"${proto}"
-  copyright_year=${tuple[0]}
-  product_path=${tuple[1]}
-  proto_file=${tuple[2]}
-  echo "Generate code from ${proto_file} to ${product_path}"
-  GOOGLE_CLOUD_CPP_ENABLE_CLOG=yes "${BAZEL_BIN_DIR}"/external/com_google_protobuf/protoc \
-    --plugin=protoc-gen-cpp_codegen="${BAZEL_BIN_DIR}"/generator/protoc-gen-cpp_codegen \
-    --cpp_codegen_out=. \
-    --cpp_codegen_opt=product_path="${product_path}" \
-    --proto_path="${BAZEL_OUTPUT_BASE}"/external/com_google_protobuf/src \
-    --proto_path="${BAZEL_OUTPUT_BASE}"/external/com_google_googleapis \
-    --cpp_codegen_opt=googleapis_commit_hash="${BAZEL_DEPS_GOOGLEAPIS_HASH}" \
-    --cpp_codegen_opt=copyright_year="${copyright_year}" \
-    "${BAZEL_OUTPUT_BASE}"/external/com_google_googleapis/"${proto_file}"
+GOOGLE_CLOUD_CPP_ENABLE_CLOG=yes "${BAZEL_BIN_DIR}"/generator/google-cloud-cpp-codegen \
+  --googleapis_commit_hash="${BAZEL_DEPS_GOOGLEAPIS_HASH}" \
+  --protobuf_proto_path="${BAZEL_OUTPUT_BASE}"/external/com_google_protobuf/src \
+  --googleapis_proto_path="${BAZEL_OUTPUT_BASE}"/external/com_google_googleapis \
+  --output_path=. \
+  --config_file=generator/generator_config.textproto
 
-  find "${product_path}" \( -name '*.cc' -o -name '*.h' \) -exec clang-format -i {} \;
-done
+io::log_yellow "clang-format generated code."
+find google/cloud -path google/cloud/bigtable -prune -o \
```

Comment:
Consider for another PR: When the generator binary writes the file perhaps it should just directly clang-format it at that point. It could write the file then call `clang-format -i...` or it could pipe its output through clang-format.